aliases/available/git.aliases.bash

```diff
@@ -103,7 +103,8 @@ alias gpatch='git format-patch -1'
 # push
 alias gp='git push'
 alias gpd='git push --delete'
-alias gpf='git push --force'
+alias gpf='git push --force-with-lease'
```

Comment:
This is a breaking change, but very important one. We should all prefer --force-with-lease over --force anyway